src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/content_key_policy.py

```diff
@@ -3,24 +3,477 @@
+    return _validate_all_conditions(
+        [token_type, token_key, token_key_type in get_tokens(), issuer, audience],
+        'Malformed content key policy token restriction.')
+
+
+def _valid_fairplay_configuration(ask, fair_play_pfx_password, fair_play_pfx, rental_and_lease_key_type, rental_duration):
+    return _validate_all_conditions(
+        [ask, fair_play_pfx_password, fair_play_pfx, rental_and_lease_key_type, rental_duration],
+        'Malformed content key policy FairPlay configuration.')
+
+
+def _valid_playready_configuration(play_ready_template):
+    if play_ready_template is None:
+        return False
+
+    def __valid_license(lic):
+        return _validate_all_conditions([lic.get('allowTestDevices') is not None,
+                                         lic.get('licenseType') in ['NonPersistent', 'Persistent'],
+                                         lic.get('contentKeyLocation') is not None,
+                                         lic.get('contentType') in ['Unspecified', 'UltraVioletDownload', 'UltraVioletStreaming'],
+                                         lic.get('playRight') is None or __valid_play_right(lic.get('playRight'))],
+                                        'Malformed PlayReady license.')
+
+    def __valid_play_right(prl):
+        return _validate_all_conditions([(prl.get('explicitAnalogTelevisionOutputRestriction') is None or
+                                          __valid_eator(prl.get('explicitAnalogTelevisionOutputRestriction'))),
+                                         prl.get('digitalVideoOnlyContentRestriction') is not None,
+                                         prl.get('imageConstraintForAnalogComponentVideoRestriction') is not None,
+                                         prl.get('imageConstraintForAnalogComputerMonitorRestriction') is not None,
+                                         prl.get('allowPassingVideoContentToUnknownOutput') in ['NotAllowed', 'Allowed',
+                                                                                                'AllowedWithVideoConstriction']],
+                                        'Malformed license PlayRight.')
+
+    def __valid_eator(eator):
+        return _validate_all_conditions([eator.get('bestEffort') is not None,
+                                         eator.get('configurationData') is not None],
+                                        'Malformed explicit analog television output restriction.')
+
+    cfg = None
+
+    try:
+        cfg = json.loads(play_ready_template)
+    except ValueError as err:
+        raise CLIError('Malformed JSON: ' + str(err))
+
+    return _validate_all_conditions(
+        [cfg.get('licenses') is not None,
+         len(cfg.get('licenses')) > 0,
+         all(__valid_license(l) for l in cfg.get('licenses'))],
+        'Malformed content key policy PlayReady configuration'
+    )
+
+
+def _validate_all_conditions(conditions, error_if_malformed):
```

Comment:
Try with validate_any instead of this. Also, make sure to validate None arguments to avoid exceptions on our side. Let the API to validate arguments missing.